modules/ocf_stats/manifests/munin.pp

```diff
@@ -42,6 +42,14 @@
   include apache::mod::fcgid
   include apache::mod::rewrite
 
+  $cname = $::host_env ? {
```

Comment:
This is a duplicate declaration

Reply:
Good catch, I got some merge conflicts when rebasing this, and it looks like I didn't resolve this correctly. I've fixed it in f3fdd5912a5dc5eafd9995412a9c5e85874dee31.